stripper/ze_postal_v1_1.cfg

```diff
@@ -0,0 +1,113 @@
+;tp avoidance spot
+add:
+{
+	"classname" "trigger_teleport"
+	"targetname" "resizeme"
+	"origin" "3011.5 -655 2075.75"
+	"UseLandmarkAngles" "0"
+	"target" "zombie_tp_3"
+	"StartDisabled" "1"
+	"spawnflags" "4097"
+	"CheckDestIfClearForPlayer" "0"
+}
+
+;Stop CTs from bullying zombies during cat fight room by doing 2 damage to that CT every shot into the zombie cage
+add:
+{
+	"classname" "point_hurt"
+	"targetname" "bully_hurt"
+	"origin" "1969 -1000 2701"
+	"Damage" "2"
+	"DamageDelay" "2"
+	"DamageRadius" "1"
+	"DamageTarget" "!activator"
+	"DamageType" "16384"
+}
+
+add:
+{
+	"classname" "func_breakable"
+	"targetname" "zombie_protector"
+	"origin" "1935 -1056 2745"
+	"spawnflags" "2048"
+	"rendermode" "10"
+	"health" "99999999"
+	"disableflashlight" "1"
+	"model" "*260"
+	"OnHealthChanged" "bully_hurt,Hurt,,0,-1"
+}
+
+add:
+{
+	"classname" "func_breakable"
+	"targetname" "zombie_protector"
+	"origin" "1989 -1056 2745"
+	"spawnflags" "2048"
+	"rendermode" "10"
+	"health" "99999999"
+	"disableflashlight" "1"
+	"model" "*261"
+	"OnHealthChanged" "bully_hurt,Hurt,,0,-1"
+}
+
+add:
+{
+	"classname" "point_template"
+	"targetname" "zombie_protection_template"
+	"origin" "1989 -1057 2745"
+	"Template01" "zombie_protector"
+	"spawnflags" "2"
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "logic_relay"
+		"targetname" "relay_after_boss"
+	}
+	insert:
+	{
+		"OnTrigger" "zombie_protector,Kill,,5,1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "func_movelinear"
+		"targetname" "boss_door2"
+		"origin" "1989 -1057 2745"
+	}
+	insert:
+	{
+		"OnFullyClosed" "zombie_protection_template,ForceSpawn,,0,1"
+	}
+}
+
+;fix doorway that was scuffed for no reason and players constantly complained about
+add:
+{
+	"classname" "func_brush"
+	"origin" "2529.5 -200 1840"
+	"angles" "0 0 0"
+	"model" "*240"
+	"rendermode" "10"
+}
```

Comment:
I was told this was intended by the mapper.

The console msgs at this part also sort of confirm this:
"OnStartTouch" "idkCommandsay ***WTF, WHY I CAN'T GET INSIDE?***51"
"OnStartTouch" "idkCommandsay ***MAPPER, YOU ARE FUCKIN' RETARD!!!***61"

Reply:
Didn't even see those map messages that show the mapper intended that, woops. However, I still think it should be changed (though obviously you have the final decision). When a large majority of the server (including the people that get through the window) complain every round about 1 spot, and it isn't due to difficulty but instead due to a scuffed csgo physics, it is frankly just not good map design. I see it no different than other stripper nerfs we have that go against how a mapper scaled a boss. In example of nerfs we already have: [Bathroom Sponge Nerf](https://github.com/gflclan-cs-go-ze/ZE-Configs/blob/aebd71e85d7914f370b7b3e10609ba9a86efc18c/stripper/ze_bathroom_v2_5.cfg#L294)
[Naruto Nerf that was just added](https://github.com/gflclan-cs-go-ze/ZE-Configs/blob/aebd71e85d7914f370b7b3e10609ba9a86efc18c/stripper/ze_Naruto_p.cfg#L1)

I would understand having something like this in a 'trolly' map like evil mansion or licciana, but on this map there are no other 'trollish' mechanics other than this window.